prime-dotnet-webapi/Controllers/BannersController.cs

```diff
@@ -49,7 +49,7 @@ public async Task<ActionResult> CreateBanner(BannerViewModel viewModel, [FromQue
         /// <param name="bannerId"></param>
         /// <param name="viewModel"></param>
         [HttpPut("{bannerId}", Name = nameof(UpdateBanner))]
-        [Authorize(Roles = Roles.PrimeSuperAdmin)]
+        [Authorize(Roles = Roles.PrimeMaintenance)]
```

Comment:
Create/delete banner probably could also be maintenance instead of super admin

Reply:
I'll have to double check tomorrow just to be sure. I asked a question about create/delete during scrum and if my memory is working the answer was that maintenance role would be more for editing text.

Reply:
Confirmed, will be adding create/delete for banners